src/firstApp/IsoscelesTriangle.java

```diff
@@ -0,0 +1,24 @@
+package firstApp;
+
+import java.util.Scanner;
+public class IsoscelesTriangle {
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+
+        int a, b, c;
+        System.out.println("Enter value A ");
+        a = scan.nextInt();
+        System.out.println("Enter value B ");
+        b = scan.nextInt();
+        System.out.println("Enter value C ");
+        c = scan.nextInt();
+        scan.close();
+
+        if (a == b) {
```

Comment:
Lets imagine that sideA != sideB but sideA == sideC - as a result triangle isosceles
Please, rewrite to handle it